grails-app/views/region/_groups.gsp

```diff
@@ -15,7 +15,8 @@
                     <g:if test="${!group.parent}">
                         <i class="fa fa-chevron-right"></i>
                     </g:if>
-                    ${group.commonName == 'ALL_SPECIES' ? message(code:"groups.all.species") : group.commonName}
+                    <g:set var="i18nName" value='${group.commonName == 'ALL_SPECIES' ? 'All Species' : group.commonName}' />
+                    <g:message code="groups.${i18nName.replaceAll("\\s+","").replaceAll(",", "")}" />
```

Comment:
Needs a default as species groups and species subgroups are subject to change. I'll add this after the merge.